src/constants.js

```diff
@@ -1,6 +1,7 @@
 export const INSUREE_MARITAL_STATUS = ["N", "W", "S", "D", "M"];
 export const FAMILY_POVERTY_STATUS = [true, false];
 export const PHOTO_STATUS = ["with", "without"];
+export const FAMILY_STATUS = ["with", "without"];
```

Comment:
These variables can be combined into a single one with a generic name.

Reply:
Same as above - if for example we have another status for photo (let's say with/without/expired) we would have to reintroduce this variable for family_status.

Reply:
Makes sense, up to you.